charts/ocis/values.yaml

```diff
@@ -108,11 +110,14 @@ registry:
   # -- Configure the service registry type.
   # Defaults to "kubernetes".
   # Can be set to "etcd" or "natsjs", then the address of etcd / nats node(s) needs to be set to `registry.nodes`.
-  type: kubernetes
+  type: nats-js-kv
   # -- Nodes of the service registry to use.
-  nodes: []
-  # nodes:
-  #   - etcd.ocis-etcd.svc.cluster.local:2379
+  nodes:
+    - nats.ocis-nats.svc.cluster.local:4222
+  # -- Username used for authentication
+  authusername: ""
+  # -- Password used for authentication
+  authpassword: ""
```

Comment:
credentials should only be stored in Secret objects. Please see the `secretRef` section how it has been done for other Secrets